tests/phpunit/includes/testcase.php

```diff
@@ -44,17 +44,22 @@ protected function assert_has_language( $object, $language, $message = '' ) {
 
 	/**
 	 * Verifies that a file exists.
-	 * Depending on the environment var `PLL_SKIP_PLUGINS_TESTS`, skips or does an assertion if the file doesn't exist.
+	 * Depending on the environment variable `GITHUB_ACTIONS`:
+	 * - If defined to `'true'` (default value in GitHub actions): does an assertion.
+	 * - If not defined to `'true'`: skips if the file doesn't exist.
+	 *
+	 * @see https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables
 	 *
 	 * @param string $path    Path to the file.
 	 * @param string $message Error message.
 	 * @return void
 	 */
 	protected static function markTestSkippedIfFileNotExists( string $path, string $message = '' ): void {
-		if ( ! getenv( 'PLL_SKIP_PLUGINS_TESTS' ) ) {
+		if ( 'true' === getenv( 'GITHUB_ACTIONS' ) ) {
```

Comment:
According to the documentation, it is *always* set to `'true'`. Do we need to check the value then?

Reply:
Not needed per say, but I prefer using booleans in `if` statements. `if ( 'true' ) {` is "only" true-ish.
Do you fear they change the returned value?

Reply:
It could, but in the other hand I agree with you on using bool value in if statements.
What about `false !== getenv( 'GITHUB_ACTIONS' )`?
`getenv()` returning `false` if `GITHUB_ACTIONS` doesn't exist at all.